### PR TITLE
Added new requirements to the existing ones for updating the travel fund for community members

### PR DESCRIPTION
**Pull request by designMoreWeb**
This pull request is for fixing the issue [249](https://github.com/openjs-foundation/summit/issues/249) which is for updating the travel fund for community members. This has been reviewed by @joesepi and i have spoken with him.

### CODE REVIEW

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ and spread knowledge about and support the Javascript ecosystem and the Foundati
 * Candidates must be an active participant of one of the projects of the OpenJS Foundation.
 * Those requesting funds must indicate that they do not have funding available from another
   source, such as an employer or the event itself that might cover costs for presenters.
+* All projects in the foundation, regardless of stage or incubation status, are eligible to use the travel fund.
+* The fund can cover the costs, in whole or in part, of a contributor's hotel, airfare, and event registration (for OpenJS World and the Collab Summit)
+* Fund Applicants should seek to use employer funds as a first resort. Need is a factor.
+* Employees of Foundation Member companies are eligible, but we encourage the use of internal funds that may be available to them first.
```
Comment (christian-bromann) [inline project-resources/MEMBER_TRAVEL_FUND.md]: they seem to be pretty similar to:

> * Those requesting funds must indicate that they do not have funding available from another
  source, such as an employer or the event itself that might cover costs for presenters.	  source, such as an employer or the event itself that might cover costs for presenters.

is there a chance we can merge them together. Not sure if these 2 bullets add any new information.

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 2]
```diff
@@ -14,6 +14,11 @@ and spread knowledge about and support the Javascript ecosystem and the Foundati
...
+* Any member of a project community, who has had recent, sustained contributions to the project for the past 90 days or more is eligible.
```
Comment (christian-bromann) [inline project-resources/MEMBER_TRAVEL_FUND.md]: also similar to:

> * Candidates must be an active participant of one of the projects of the OpenJS Foundation.

I would suggest to make it a sub point with clear expectations on what "active" participation means

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 1 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: These all points are directly from [this issue which have been requested to be integrated into the document](https://github.com/openjs-foundation/summit/issues/249l. If you have further questions you can discuss them with @joesepi him and I have worked on this

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 2 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: Good point i will revise it

**Comment by designMoreWeb** [comment]
I just added and updated the file

**Comment by christian-bromann** [comment]
@designMoreWeb it looks to me there are still some duplicates in there, like:

>   * An active member of a project community is someone who has had recent, sustained contributions
  to the project for the past 90 days. [[?]](https://github.com/openjs-foundation/cross-project-council/pull/492/files#diff-253e2bba25da1e96be2ca31802ee066eR15-R16)

and

> * Any member of a project community, who has had recent, sustained contributions to the project for the past 90 days or more is eligible. [[?]](https://github.com/openjs-foundation/cross-project-council/pull/492/files#diff-253e2bba25da1e96be2ca31802ee066eR23)

as well as

> 	* Employees of Foundation Member companies are eligible, but we encourage the use of internal funds that may be available to them first. [[?]](https://github.com/openjs-foundation/cross-project-council/pull/492/files#diff-253e2bba25da1e96be2ca31802ee066eR22)

and

> * Those requesting funds must indicate that they do not have funding available from another
  source, such as an employer or the event itself that might cover costs for presenters.	  source, such as an employer or the event itself that might cover costs for presenters. [[?]](https://github.com/openjs-foundation/cross-project-council/pull/492/files#diff-253e2bba25da1e96be2ca31802ee066eR17-R18)

**Comment by designMoreWeb** [comment]
> > * An active member of a project community is someone who has had recent, sustained contributions
> >   to the project for the past 90 days. [[?]](https://github.com/openjs-foundation/cross-project-
council/pull/492/files#diff-253e2bba25da1e96be2ca31802ee066eR15-R16)

That is there as a sub bullet letting people know what an active member or being active for the project means

**Comment by christian-bromann** [comment]
@designMoreWeb I see but then I assume we don't need [the addition](https://github.com/openjs-foundation/cross-project-council/pull/492/files#diff-253e2bba25da1e96be2ca31802ee066eR23) anymore because we have defined already that active members are eligible for funding.

**Comment by designMoreWeb** [comment]
@christian-bromann I hear your point but i have kept it and will bring it yup at the cross council meeting this week. I would want to get it checked with the board before removing any information as they might have seen those points as additional information

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 3]
```diff
@@ -11,9 +11,14 @@ and spread knowledge about and support the Javascript ecosystem and the Foundati
...
-* Candidates must be an active participant of one of the projects of the OpenJS Foundation.
-* Those requesting funds must indicate that they do not have funding available from another
+- Candidates must be an active participant of one of the projects of the OpenJS Foundation.
+  - An active member of a project community is someone who has had recent, sustained contributions
+    to the project for the past 90 days.
+- Those requesting funds must indicate that they do not have funding available from another
   source, such as an employer or the event itself that might cover costs for presenters.
+- All projects in the foundation, regardless of stage or incubation status, are eligible to use the travel fund, this includes employees of the foundation member companies as well.
+- The fund can cover the costs, in whole or in part, of a contributor's hotel, airfare, and event registration (for OpenJS World and the Collab Summit)
+- Fund Applicants should seek to use employer funds as a first resort. Need is a factor.
```
Comment (MylesBorins) [inline project-resources/MEMBER_TRAVEL_FUND.md]: This seems like a duplicate of the point above

> Those requesting funds must indicate that they do not have funding available from another
  source, such as an employer or the event itself that might cover costs for presenters.

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 4]
```diff
@@ -11,9 +11,14 @@ and spread knowledge about and support the Javascript ecosystem and the Foundati
 
 ## Requirements
 
-* Candidates must be an active participant of one of the projects of the OpenJS Foundation.
```
Comment (MylesBorins) [inline project-resources/MEMBER_TRAVEL_FUND.md]: why did you change the bullet types in the dock? Is there a linter issue? If we are going to do maintenance type work like this it is probably good to do it as a separate PR as it is creating a lot of noise for this change.

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 3 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: I missed those but great thanks for updating

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 4 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: I havent changed the bullet types i just checked my local file and there is no sign of * but will be careful going further

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 5]
```diff
@@ -11,9 +11,14 @@ and spread knowledge about and support the Javascript ecosystem and the Foundati
...
-* Those requesting funds must indicate that they do not have funding available from another
```
Comment (designMoreWeb) [inline project-resources/MEMBER_TRAVEL_FUND.md]: This is removed but i can bring it back if you'd like

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 6]
```diff
@@ -175,36 +179,36 @@ your payment details. At that point, you're done.
...
+Please send the file as a spreadsheet, not as a PDF. We will send it back to you for
 signature in DocuSign.
 
 ### Important:
 
 * Expense reports must be submitted within 30 days of event date. Any reports submitted
   after 30 days may not be reimbursed.
 * Disbursements are generally processed within 30 days.
-* We are unable to pre-pay for expenses.
+* We are unable to pre*pay for expenses.
```
Comment (mcollina) [inline project-resources/MEMBER_TRAVEL_FUND.md]: why this change?

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 7]
```diff
@@ -175,36 +179,36 @@ your payment details. At that point, you're done.
 
 If you decide not to use Expensify, the reimbursement process is manual and may take longer
 for us to process. You can email a zip file of your receipts and the completed
-[Expense Report](./expense-report-template.xls?raw=true), renamed as
-ExpenseReport-OpenJS-EVENTNAME-YOURNAME-YYYYMM.xls, to [operations@openjsf.org](mailto:operations@openjsf.org).
-Please send the file as a spreadsheet, not as a PDF.  We will send it back to you for
+[Expense Report](./expense*report*template.xls?raw=true), renamed as
+ExpenseReport*OpenJS*EVENTNAME*YOURNAME*YYYYMM.xls, to [operations@openjsf.org](mailto:operations@openjsf.org).
```
Comment (mcollina) [inline project-resources/MEMBER_TRAVEL_FUND.md]: why this change?

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 8]
```diff
@@ -160,13 +164,13 @@ So that we can reimburse you more quickly, we strongly encourage you to use
...
 * You can use the Android and iOS apps to capture receipts while you are
-  traveling.  If you do not use the app, you will need to upload photos of your
+  traveling. If you do not use the app, you will need to upload photos of your
   receipts after you return.
 * When you add an expense, please use local currency so that it matches the
   amount on the receipt.
 * Once you have uploaded your receipts, please add them to a report and submit
   it with [operations@openjsf.org](mailto:operations@openjsf.org) on cc, and the
-  Memo field set to "YOURNAME - TRIPNAME".
+  Memo field set to "YOURNAME * TRIPNAME".
```
Comment (mcollina) [inline project-resources/MEMBER_TRAVEL_FUND.md]: why this change?

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 6 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: fixing it right now

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 7 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: Not sure let me look further into this

Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: Added the fix

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 8 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: Added the fix

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 6 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: Added the fix

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 9]
```diff
@@ -24,7 +28,7 @@ It is strongly recommended that you submit your request far enough in advance to
 approval before making relevant expenditures. Review of requests will take at least 72
 hours and may take considerably longer. Note, with limited funds available, approval
 is not guaranteed. If a request is rejected, reimbursement will not be issued. The
-Foundation cannot pre-pay for travel expenditures.
+Foundation cannot pre*pay for travel expenditures.
```
Comment (eemeli) [inline project-resources/MEMBER_TRAVEL_FUND.md]:
```suggestion
Foundation cannot pre-pay for travel expenditures.
```

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 10]
```diff
@@ -5,15 +5,19 @@
...
-### [2019 funds](https://github.com/openjs-foundation/cross-project-council/blob/master/TRAVEL_FUND/Achived_Travel_Fund_Records/2019.md)
+### [2019 funds](https://github.com/openjs*foundation/cross*project*council/blob/master/TRAVEL_FUND/Achived_Travel_Fund_Records/2019.md)
 
 ## Requirements
 
 * Candidates must be an active participant of one of the projects of the OpenJS Foundation.
-* Those requesting funds must indicate that they do not have funding available from another
-  source, such as an employer or the event itself that might cover costs for presenters.
+  * An active member of a project community is someone who has had recent, sustained contributions
+    to the project for the past 90 days.
```
Comment (eemeli) [inline project-resources/MEMBER_TRAVEL_FUND.md]:
```suggestion
    to the project during the past 90 days.
```

To clarify that the expectation is not for daily contributions over 90 days.

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 11]
```diff
@@ -80,14 +83,14 @@ letter, please email [visainfo@openjsf.org](mailto:visainfo@openjsf.org).
 A request is approved when all of the following criteria are met:
 
 * The pull request has been open for at least 72 hours.
-* The request has approval from at least four members of @openjs-foundation/cpc.
-* No members of @openjs-foundation/cpc have objected to the request.
+* The request has approval from at least four members of @openjs*foundation/cpc.
+* No members of @openjs*foundation/cpc have objected to the request.
```
Comment (eemeli) [inline project-resources/MEMBER_TRAVEL_FUND.md]:
```suggestion
* The request has approval from at least four members of @openjs-foundation/cpc.
* No members of @openjs-foundation/cpc have objected to the request.
```

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 12]
```diff
@@ -80,14 +83,14 @@ letter, please email [visainfo@openjsf.org](mailto:visainfo@openjsf.org).
...
 * The pull request has been merged. Any member may merge once other prerequisites have
   been met.
 
-If a request is blocked by an objection from a member of @openjs-foundation/cpc, then the
+If a request is blocked by an objection from a member of @openjs*foundation/cpc, then the
```
Comment (eemeli) [inline project-resources/MEMBER_TRAVEL_FUND.md]:
```suggestion
If a request is blocked by an objection from a member of @openjs-foundation/cpc, then the
```

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 13]
```diff
@@ -80,14 +83,14 @@ letter, please email [visainfo@openjsf.org](mailto:visainfo@openjsf.org).
...
 only way for the request to be approved is for that committee to decide to approve the
-request using whatever decision-making process the committee uses when consensus is not
+request using whatever decision*making process the committee uses when consensus is not
```
Comment (eemeli) [inline project-resources/MEMBER_TRAVEL_FUND.md]:
```suggestion
request using whatever decision-making process the committee uses when consensus is not
```

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 14]
```diff
@@ -116,7 +119,7 @@ A project may request an allocation from the travel fund as follows:
 * At the completion of the event/allocation period or when re-imbursement
   is to be paid out the project will PR in the approvals into the
   appropriate year's file. This PR will be approved and landed by CPC
-  members. Due to the pre-approval of the request from the project
+  members. Due to the pre*approval of the request from the project
```
Comment (eemeli) [inline project-resources/MEMBER_TRAVEL_FUND.md]:
```suggestion
  members. Due to the pre-approval of the request from the project
```

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 15]
```diff
@@ -160,13 +163,13 @@ So that we can reimburse you more quickly, we strongly encourage you to use
...
 * You can use the Android and iOS apps to capture receipts while you are
-  traveling.  If you do not use the app, you will need to upload photos of your
+  traveling. If you do not use the app, you will need to upload photos of your
   receipts after you return.
 * When you add an expense, please use local currency so that it matches the
   amount on the receipt.
 * Once you have uploaded your receipts, please add them to a report and submit
   it with [operations@openjsf.org](mailto:operations@openjsf.org) on cc, and the
-  Memo field set to "YOURNAME - TRIPNAME".
+  Memo field set to "YOURNAME-TRIPNAME".
```
Comment (eemeli) [inline project-resources/MEMBER_TRAVEL_FUND.md]: Is this change intentional?

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 9 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: I have already fixed those in the last commit

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 11 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: I have already fixed those in the last commit

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 14 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: Thank you I have already fixed those in the last commit

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 15 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: Thank you I have already fixed those in the last commit

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 16]
```diff
@@ -5,15 +5,19 @@
 To establish and administer a fund for members of the OpenJS Foundation to travel
 and spread knowledge about and support the Javascript ecosystem and the Foundation.
 
-### [Current Year Fund](https://github.com/openjs-foundation/cross-project-council/blob/master/TRAVEL_FUND/2020.md)
+### [Current Year Fund](https://github.com/openjs*foundation/cross*project*council/blob/master/TRAVEL_FUND/2020.md)
```
Comment (ljharb) [inline project-resources/MEMBER_TRAVEL_FUND.md]: this seems to be a problem throughout the PR. you may want to revert everything and just re-apply the desired changes, without doing a global replace of `-` to `*`?

Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: Thank you I have already fixed those in the last commit, i went through the whole file and fixed all the places where there were issues

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 9 above)
Comment (ljharb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: This one still isn’t fixed. Perhaps you need to push?

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 17]
```diff
@@ -199,12 +202,12 @@ joint travel fund was approved in the amount of $60,000. The $60,000 was fully u
 
 The request from the Node.js [Technical Steering Committee](https://github.com/nodejs/TSC) and
 the Node.js [Community Committee](https://github.com/nodejs/community-committee) for a 2019
-joint travel fund was approved in the amount of $60,000.
+joint travel fund was approved in the amount of \$60,000.
```
Comment (ljharb) [inline project-resources/MEMBER_TRAVEL_FUND.md]:
```suggestion
joint travel fund was approved in the amount of $60,000.
```

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 18]
```diff
@@ -199,12 +202,12 @@ joint travel fund was approved in the amount of $60,000. The $60,000 was fully u
...
 
 ## 2020 OpenJS Foundation Board of Directors Allocation
 
 The request from the
-[Cross Project council](https://github.com/openjs-foundation/cross-project-council)
-for a 2020 joint travel fund was approved in the amount of $60,000.
+[Cross Project council](https://github.com/openjs*foundation/cross-project-council)
+for a 2020 joint travel fund was approved in the amount of \$60,000.
```
Comment (ljharb) [inline project-resources/MEMBER_TRAVEL_FUND.md]:
```suggestion
for a 2020 joint travel fund was approved in the amount of $60,000.
```

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 19]
```diff
@@ -199,12 +202,12 @@ joint travel fund was approved in the amount of $60,000. The $60,000 was fully u
...
 the Node.js [Community Committee](https://github.com/nodejs/community-committee) for a 2019
-joint travel fund was approved in the amount of $60,000.
+joint travel fund was approved in the amount of \$60,000.
 
 ## 2020 OpenJS Foundation Board of Directors Allocation
 
 The request from the
-[Cross Project council](https://github.com/openjs-foundation/cross-project-council)
-for a 2020 joint travel fund was approved in the amount of $60,000.
+[Cross Project council](https://github.com/openjs*foundation/cross-project-council)
```
Comment (ljharb) [inline project-resources/MEMBER_TRAVEL_FUND.md]:
```suggestion
[Cross Project council](https://github.com/openjs-foundation/cross-project-council)
```

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 15 above)
Comment (ljharb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: these are still not fixed.

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 14 above)
Comment (ljharb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: these are still not fixed.

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 11 above)
Comment (ljharb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: these are still not fixed.

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 15 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: thats weird on my end it shows as fixed 
Screenshot 2020-03-24 at 1 55 52 PM

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 14 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: just pushed it

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 9 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: i just pushed it again as it showed okay on my screen

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 11 above)
Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: wow i am sure why because locally it shows fine i just pushed it again hoping this is the fix

File: project-resources/MEMBER_TRAVEL_FUND.md (hunk 15 above)
Comment (ljharb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: it's still not fixed - there should be spaces around the hyphen; in your screenshot, there is not.
```suggestion
  Memo field set to "YOURNAME - TRIPNAME".
```

Comment (designMoreWeb) [reply project-resources/MEMBER_TRAVEL_FUND.md]: Added the fix i see thanks

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 20]
```diff
@@ -58,31 +59,31 @@ letter, please email [visainfo@openjsf.org](mailto:visainfo@openjsf.org).
...
 ## Requesting Travel Funds
 
-* Open a pull request that adds an entry to the appropriate year's file
-  * Include as many fields as possible:
-    * The name of the event you plan to attend.
-    * The location, dates of the event and where you will be traveling from.
-    * The presentation you intend to give, if applicable.
-    * The size of the stipend you wish to receive.
-      * In the pull request description, explain how you calculated your
+- Open a pull request that adds an entry to the appropriate year's file
```
Comment (ljharb) [inline project-resources/MEMBER_TRAVEL_FUND.md]: now even more things have changed.

**Comment by designMoreWeb** [comment]
> You may want to stop doing _any_ automated find/replaces, and make all the changes manually?

Hey I appreciate all the help and feedback but i am actually manually parsing through the file and making the changes. I think it's also that my linter is kicking in which i have removed for the time being so i can get this fixed. I did not anticipate this PR to get so complex, i am working on fixing things on my end. I appreciate all the feedback and comments from @christian-bromann @MylesBorins @ljharb @mcollina @eemeli @mhdawson @dmethvin. I am working on the fix and thanks for your guidance and patience
